pages/api/fiscal-sponsorship/apply.js

```diff
@@ -61,7 +61,8 @@ export default async function handler(req, res) {
         '(Adults) Political Activity': data.eventPoliticalActivity,
         '(Adults) Annual Budget': parseInt(data.eventAnnualBudget),
         'HCB ID': r.id,
-        'Referral Code': data.referralCode
+        'Referral Code': data.referralCode,
+        Tubs: data.referralCode ? 'GFGS' : ''
```

Comment:
In the future, let's have TUBs generate application links that include a prefilled value for the `Tubs` value. This includes GFGS.

The advantage is that we won't have TUB-specific code.